app/models/project.rb

```diff
@@ -37,4 +39,20 @@ def add_default_repayment_rate
   def add_default_repayment_begin
     self.repayment_begin = requested_by + 90
   end
+
+  def add_default_amount_needed
+    self.amount_needed = price
+  end
+
+  def calculate_amount_needed(loan)
```

Comment:
Unused method argument - `loan`. If it's necessary, use `_` or `_loan` as an argument name to indicate that it won't be used. You can also write as `calculate_amount_needed(*)` if you want the method to accept any arguments but don't care about them.